mmv1/overrides/runner.rb

```diff
@@ -102,16 +101,10 @@ def build_resource(old_resource, res_override, override_classes)
         variables = (old_resource.instance_variables + res_override.instance_variables).uniq
         variables.reject { |o| %i[@properties @parameters].include?(o) }
                  .each do |var_name|
-          # Check if the variable is defined on the original api but wasn't explicitly specified
-          # on the override. This stops override defaults (such as custom_code) from clobbering
-          # properties that exist on the api already
-          if old_resource.instance_variables.include?(var_name) \
-             && !explicit_overrides.include?(var_name)
+          if res_override[var_name].nil?
```

Comment:
AIUI no longer relevant, since Terraform doesn't use defaults for overrides. Honestly, given this had no diffs and we're gonna delete this file within the month (once overrides are gone completely) I didn't validate _too_ hard.